packages/web/.eslintrc.json

```diff
@@ -15,7 +15,7 @@
       }
     ],
     "@next/next/no-img-element": "off",
-    "react-hooks/exhaustive-deps": "off",
+    // "react-hooks/exhaustive-deps": "off",
```

Comment:
So should this setting be on?

Reply:
The time is on by default, I will delete the comment.